other/materials_designer/specific_examples/defect_point_interstitial_tin_oxide.ipynb

```diff
@@ -0,0 +1,288 @@
+{
```

Comment:
<p>the list needs reformatting</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>